### PR TITLE
Updating file server connector annotation syntax

### PR DESCRIPTION
**Pull request by amjadhifthikar**

### CODE REVIEW

File: modules/ballerina-native/src/main/ballerina/ballerina/net/file/annotation.bal [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package ballerina.net.file;
+
+annotation FileSource attach service {
+    string protocol;
```
Comment (appellj) [inline modules/ballerina-native/src/main/ballerina/ballerina/net/file/annotation.bal]: Is protocol still valid?

File: docs/server-connectors/file-server.md [hunk 2]
```diff
@@ -93,12 +93,11 @@ name, size, lastModifiedTimestamp</td>
 #### Example: 
 
 ```
-@Source (
-  protocol = "file",
```
Comment (appellj) [inline docs/server-connectors/file-server.md]: The table still says protocol is required, but it's not in the examples.

File: modules/ballerina-native/src/main/ballerina/ballerina/net/file/annotation.bal (hunk 1 above)
Comment (amjadhifthikar) [inline modules/ballerina-native/src/main/ballerina/ballerina/net/file/annotation.bal]: Protocol is not needed now. Will remove it and send a PR.

File: docs/server-connectors/file-server.md (hunk 2 above)
Comment (amjadhifthikar) [inline docs/server-connectors/file-server.md]: Fixed it in pull request #2369

**Pull request merged**